### PR TITLE
ipv6 parsing

### PR DESCRIPTION
**Pull request by azlux**
try to solve #426 issue.
I hope the default offset is 0

---
This change is [Reviewable](https://reviewable.io/reviews/exa-networks/exabgp/428)

### CODE REVIEW

**Comment by thomas-mangin** [comment]
Thank you very much for the patch, much appreciated. I can however not merge it as you have have some IPv6 in the form ( like :ffff:192.0.2.128 ), so the first test need to test for the presence of : and not .

**Comment by azlux** [comment]
I can do it for the specific format. But 0000:0000:0000:0000:0000:0000:127.127.127.127 is a [depreciate format](https://tools.ietf.org/html/rfc4291#section-2.5.5.1) now.
Do you really want to use it ?

File: lib/exabgp/configuration/flow/parser.py [hunk 1]
```diff
@@ -50,22 +50,30 @@ def flow (tokeniser):
...
 
 
 def destination (tokeniser):
 	data = tokeniser()
-	if data.count('/') == 1:
+	if data.count('.') == 3 and data.count(':') == 0 :
 		ip,netmask = data.split('/')
 		raw = ''.join(chr(int(_)) for _ in ip.split('.'))
 		yield Flow4Destination(raw,int(netmask))
 		return
```
Comment (thomas-mangin) [inline lib/exabgp/configuration/flow/parser.py]: If you can please remove that return - I will merge :-)

**Comment by thomas-mangin** [comment]
Also .... sorry ... once the return is gone can you please merge all the commit in one. It can be done with git rebase -i HEAD~5  using "s" for squashing the commit together (keep the first one and use s for the following).

**Comment by azlux** [comment]
first time I use the remake function.Hope it's work.
What do you mean "the return" into your previous comment ? (I've suppose you was talking about the additional space)

**Comment by thomas-mangin** [comment]
rebase - just gave you the syntax to run after another commit
return - there is/was a left over useless line with only a "return" in the code

**Comment by thomas-mangin** [comment]
Thank you very much for the patch.

**Pull request merged**